examples/main/stylesheets/main.scss

```diff
@@ -245,9 +245,6 @@ hr.short {
   bottom: 24px;
   color: $white-40;
 }
-.overlays {
-  cursor: crosshair;
-}
```

Comment:
Having `crosshair` as the default cursor across the examples & website seemed like a mistake. It overrides the default cursor behavior of the library (and doesn't look great IMO)